content/resources/user/gen3-client.md

```diff
@@ -118,15 +164,16 @@ Downloading dg.7519/b4642430-8c6e-465a-8e20-97c458439387...
 ```
 
 * * *
-## 5) Provide a Manifest File for Bulk Download
+## 6) Provide a Manifest File for Bulk Download
 * * *
-A download manifest can be generated using the data commons' "Exploration" tool. After a cohort has been selected, clicking the "Manifest" button will create the manifest for the selected files. The gen3-client will download all the files in the provided manifest using the `gen3-client download-manifest` command.
+A download manifest can be generated using a Gen3 data common's "Exploration" tool. After a cohort has been selected, clicking the "Manifest" button will create the manifest for the selected files. The gen3-client will download all the files in the provided manifest using the `gen3-client download-manifest` command.
```

Comment:
Might need more details here on where to find the Exploration tool

".... clicking the "Download Manifest" button..."